packages/py3_pip.rb

```diff
@@ -11,8 +11,8 @@ class Py3_pip < Python
   binary_compression 'tar.zst'
 
   binary_sha256({
-    aarch64: '6723755da724e0c6b3fada4b216c558e8ec465dcc60c1bbdd0df9e34cd6b456f',
-     armv7l: '6723755da724e0c6b3fada4b216c558e8ec465dcc60c1bbdd0df9e34cd6b456f',
+    aarch64: '0fda99f7d732d18a6e35ed4f646fafc712e1f83ac1d5f49ba89612ff19a70d11',
+     armv7l: '0fda99f7d732d18a6e35ed4f646fafc712e1f83ac1d5f49ba89612ff19a70d11',
        i686: 'c9e3e1efd297e6705c4dcf29ce71c99514372fba0625feb488066f9c1df538f7',
      x86_64: 'febc4b731207ca89c0413a0c13e74b8eba3ffff9407498e790d3b0e506c58ae5'
```

Comment:
Missing updated binaries?

Reply:
Note that the existing hashes for other architectures are valid:
```
sha256sum /usr/local/tmp/packages/py3_pip-25.3-py3.14-chromeos-i686.tar.zst
c9e3e1efd297e6705c4dcf29ce71c99514372fba0625feb488066f9c1df538f7  /usr/local/tmp/packages/py3_pip-25.3-py3.14-chromeos-i686.tar.zst
```